packages/itmat-setup/src/databaseSetup/seed/organisations.ts

```diff
@@ -0,0 +1,281 @@
+export const seedOrganisations = [{
```

Comment:
We don't want to hardcode these in the codebase.
It's good to have them, but with the exception of maybe a single "System" organisation used for the first user created, this should be removed. We will be adding them to our deployment manually.

Reply:
@fguitton  do you think I should create a UI for adding an organisation? or just create an API and we can add more organisations calling that API (e.g., execting a graphql mutation)?

Reply:
That is not necessary for now, Ultimately we will want that, but right now we just need to be able to have support for these organisations